accounts/views.py

```diff
@@ -729,6 +729,39 @@ def attribution(request):
     return render(request, 'accounts/attribution.html', tvars)
 
 
+@login_required
+def download_attribution(request):
+    content = {"csv": {"style": "csv", "pattern": "%s,%s,%s,%s\n"},
+               "txt": {"style": "plain", "pattern": "%s: %s by %s | License: %s\n"}}
+
+    qs_sounds = Download.objects.annotate(download_type=Value("sound", CharField()))\
+        .values('download_type', 'sound_id', 'sound__user__username', 'sound__original_filename',
+                'license__name', 'sound__license__name', 'created').filter(user=request.user)
+    qs_packs = PackDownload.objects.annotate(download_type=Value("pack", CharField()))\
+        .values('download_type', 'pack_id', 'pack__user__username', 'pack__name', 'pack__name',
+                'pack__name', 'created').filter(user=request.user)
+    # NOTE: see the above view, attribution.
+    qs = qs_sounds.union(qs_packs).order_by('-created')
+
+    download = request.GET.get("dl", "")
+    if download in ["csv", "txt"]:
+        response = HttpResponse(content_type='text/%s' % content[download]["style"])
+        response['Content-Disposition'] = 'attachment; filename="attribution.%s"' % download
```

Comment:
it would be good to include the username and current date/time in this filename. We can make it in a separate variable to make this line simpler